### PR TITLE
Fix time-series torque widget

### LINKED ISSUE #6: Change Torque time-series widget behavior on range selection (2nd iteration)
The current behavior when a range is selected in a torque time-series widget (as of #786) is to animate the time-slider within the range. When a range is selected the behavior should be changed to:

Awaits https://github.com/CartoDB/cartodb.js/issues/905

- vis/map:
  - [x] all points within the selected time-series should be visible on the map
  - [x] animation must be stopped
- widget:
  - [x] the time-slider should not be available
  - [x] the controls (play/pause) should not be available
  - [x] instead of time-step info the selected range should be displayed (something like _"selected from `<start-time>` to `<end-time>`"_)
  - [x] a button to clear the selected range should be available
  - [x] histogram size/bins should remain after first render of data, even if other filters (e.g. bbox) and/or range selection is changed

### LINKED ISSUE #34: Fix torque time-series widget to update range-filter for selected range
In #33 I've [commented it out](https://github.com/CartoDB/deep-insights.js/commit/80b2388ada2e3e8a7d9dbab1bf4a71e17d455415#diff-c6b5f506b4e37c94f215669100b4c54bR94) since it forces the torque-layer to be rendered in faulty state (restarts animation as single step from the start again). Will need to investigate more in-depth why this happens and find a proper solution.

cc @rochoa

### PR DESCRIPTION
**Pull request by viddo**
Resolves #6 
Resolved #34 
hopefully once-and-for-all.

fix-time-series

- [x] Depends on https://github.com/CartoDB/cartodb.js/pull/1003, once merged I'll update the dependency ref.

@rochoa can you review? cc @alonsogarciapablo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   ],
   "dependencies": {
     "backbone": "1.2.3",
-    "cartodb.js": "CartoDB/cartodb.js#f0bd380",
+    "cartodb.js": "CartoDB/cartodb.js#5aa87cf",
```
Comment (rochoa) [inline package.json]: Are we still using a fixed version for a reason? I mean, couldn't we use `"CartoDB/cartodb.js#v4"`?

**Comment by rochoa** [comment]
About examples... I wouldn't try to rely in staging anymore, and I would start doing examples with documentation production account.

**Comment by rochoa** [comment]
That should be addressed here: https://github.com/CartoDB/deep-insights.js/issues/80

**Comment by rochoa** [comment]
LGTM. I'm :ship: :it: as my only worry is not a blocker.

File: package.json (hunk 1 above)
Comment (viddo) [inline package.json]: Considering cartodb.js v4 is not versioned and things are changing fast, we have experienced that this repo failed on occasions due to incompatibilities, so we changed back to using "commit-based" versioning for the time-being, to make sure things are always working.

**Pull request merged**